README.md

```diff
@@ -1 +1,64 @@
-# Open edX LTI Store (openedx-ltistore)
+# Openedx LTI Store
+
+A plugin for openedx-platform to store LTI configurations centrally. This allows course creators to add [LTI components](https://edx.readthedocs.io/projects/edx-partner-course-staff/en/latest/exercises_tools/lti_component.html) without having to enter the details like secrets, keys and URLs everytime the component block is created.
+
+## Development
+
+The development instructions are written with the [Open edX Devstack](https://edx.readthedocs.io/projects/open-edx-devstack/en/latest/index.html) as reference.
+
+1. Clone the repostiory to `<devstack_root>/src/` directory
+    ```
+    cd <devstack_root>/src/
+    git clone git@github.com:open-craft/openedx-ltistore.git
+    ```
+2. Install the plugin inside the container
+    ```
+    cd <devstack_root>/devstack/
+    make studio-shell
+    pip install -r /edx/src/openedx-ltistore
+    ```
+3. Add the `openedx_ltistore` to the installed apps. Edit the file `edx-platform/cms/envs/devstack.py`
+    ```diff
+    -INSTALLED_APPS += ['debug_toolbar']
+    +INSTALLED_APPS += ['debug_toolbar', 'openedx_ltistore']
```

Comment:
This should not be needed if this has the proper hooks in place. Pluggable apps do this automatically.
Checkout the code in the proctoring app:

- https://github.com/openedx/edx-proctoring/blob/6a21189c6e8c2ac421fe0e9755daf9a7a7a56916/setup.py#L93
- https://github.com/openedx/edx-proctoring/blob/6a21189c6e8c2ac421fe0e9755daf9a7a7a56916/edx_proctoring/apps.py#L60

We also have a few client pluggable apps in GitLab, take a look at them as well.

Reply:
@giovannicimolin Thank you for the pointers. Implemented the plugin mechanism based on the Proctering app.